tracer.go

```diff
@@ -525,7 +524,6 @@ type tracerConfig struct {
 	metricsInterval         time.Duration
 	logger                  WarningLogger
 	metricsGatherers        []MetricsGatherer
-	contextSetter           stacktrace.ContextSetter
 	preContext, postContext int
```

Comment:
Do we still need preContext/postContext? I think we can get rid of all related config and env vars?

Reply:
seems like it, removed :magic_wand: